src/python/pants/backend/experimental/java/register.py

```diff
@@ -45,6 +45,7 @@ def rules():
         *coursier_fetch.rules(),
         *java_parser.rules(),
         *java_parser_launcher.rules(),
+        *resources.rules(),
```

Comment:
These should also be exported in `src/python/pants/backend/experimental/scala/register.py`.